fill_er_up.rb

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env ruby
+
+# Load Rails
+ENV['RAILS_ENV'] = ARGV[0] || 'development'
+DIR = File.dirname(__FILE__)
+require DIR + '/../manageiq/config/environment'
```

Comment:
Seems the above assumes the script is located inside manageiq.
I think it should be based on CWD